src/Sylius/Behat/Page/Admin/Payment/IndexPage.php

```diff
@@ -33,6 +33,20 @@ public function getPaymentStateByOrderNumber(string $orderNumber): string
         return $this->getField($orderNumber, 'state')->getText();
     }
 
+    public function isPaymentOfNumberInCorrectPositionInTheList(string $orderNumber, int $position): bool
+    {
+        $result = $this->getElement('payment_of_number_in_correct_position_in_the_list', [
+            '%position%' => $position,
+            '%orderNumber%' => $orderNumber,
+        ]);
+
+        if ($result !== null) {
+            return true;
+        }
+
+        return false;
```

Comment:
Could be written in one line
```suggestion
        return $result !== null;
```